server/paymentProviders/paypal/webhook.ts

```diff
@@ -83,9 +83,6 @@ const loadSubscriptionForWebhookEvent = async (req: Request, subscriptionId: str
 };
 
 async function handleSaleCompleted(req: Request): Promise<void> {
-  // TODO During the internal testing phase, we're logging all webhooks events to make debugging easier
-  logger.info(`PayPal webhook (PAYMENT.SALE.COMPLETED): ${JSON.stringify(req.body)}`);
```

Comment:
Took the opportunity to remove these logs, storing the object in `data` is now enough